docs/resources/Supported_Devices.md

```diff
@@ -0,0 +1,174 @@
+* NCDHW refers to image sequence data layout.
+
+Abbreviations in the support tables are as follows:
+
+* N: Number of images in a batch
+* D: Depth. Depend on model it could be spatial or time dimension
+* H: Number of pixels in the vertical dimension
+* W: Number of pixels in the horizontal dimension
+* C: Number of channels
+
+CHW, NC, C  - Tensor memory layout.
+For example, the CHW value at index (c,h,w) is physically located at index `(c*H+h)*W+w`, for others by analogy.
+
+
+Supported Precision
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+==================  ===========================  ============================  ==========================
+ Plugin              FP32                         FP16                          I8                              
+==================  ===========================  ============================  ========================== 
+ CPU plugin          Supported and preferred      Supported                     Supported                                                        
+ GPU plugin          Supported                    Supported and preferred       Supported                                                        
+ GNA plugin          Supported                    Supported                     Not supported                                                    
+ Arm® CPU plugin     Supported and preferred      Supported                     Supported (partially)                                            
+==================  ===========================  ============================  ==========================
+
+
+For :doc:`Multi-Device <openvino_docs_OV_UG_Running_on_multiple_devices>` and
+:doc:`Heterogeneous <openvino_docs_OV_UG_Hetero_execution>` executions, the supported models formats depends 
+on the actual underlying devices. *Generally, FP16 is preferable as it is most ubiquitous and performant*.
+
+Supported Input Precision
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+=================  ===========  ===============  ==============  ===============  ==============  =================
+ Plugin             FP32         FP16             U8              U16              I8              I16            
+=================  ===========  ===============  ==============  ===============  ==============  =================
+ CPU plugin         Supported    Supported        Supported       Supported        Supported       Supported      
+ GPU plugin         Supported    Supported*       Supported*      Supported*       Not supported   Supported*    
+ GNA plugin         Supported    Not supported    Supported       Not supported    Supported       Supported      
+ Arm® CPU plugin    Supported    Supported        Supported       Supported        Not supported   Not supported  
+=================  ===========  ===============  ==============  ===============  ==============  =================
+
+'*' - Supported via ``SetBlob`` only, ``GetBlob`` returns FP32
```

Comment:
I think this line is outdated.
1. It refers to the old API
2. 2.0 API expects that FP32 precision to be used for all floating-point inputs/outputs. Usage of another data type requires explicit preprocessing pipeline setup and after that both get_tensor() and set_tensor() works with same data type.